src/snmp.c

```diff
@@ -449,6 +451,48 @@ static int csnmp_config_add_host_version(host_definition_t *hd,
   return 0;
 } /* int csnmp_config_add_host_address */
 
+static int csnmp_config_add_host_timeout(host_definition_t *hd,
```

Comment:
Please use `cf_util_get_int()` from `"configfile.h"` instead:

```c
if (strcasecmp("Timeout", option->key) == 0)
  status = cf_util_get_int(option, &hd->timeout);
```

Or, better yet, change `host_definition_t.timeout` to type `cdtime_t` and use `cf_util_get_cdtime()`.